app/routes/goal_routes.py

```diff
@@ -1 +1,68 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, make_response, Response
+from app.models.goal import Goal
+from app.models.task import Task
+from .. import db
+from .route_utilities import validate_model, create_model, get_models_with_filters
```

Comment:
You import this method but don't use it. 

If you're not going to use an imported object then it should be removed

```suggestion
from .route_utilities import validate_model, create_model
```

Reply:
Fixed as requested (refactored get_all_goals).